### ISSUE TITLE
Missing pictures listing on the bottom of the dialog doesn't dynamically update

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#989]
I appreciate that the picture dialog now notifies users if pictures requested by the project are not found by PTXprint. This is very helpful.

![Image](https://github.com/user-attachments/assets/05635d81-0b18-4cbf-b3ab-4aa7dbbf475d)

This list is apparently only updated after we hit the "Print (Make PDF)" button. For example, if I uncheck the "Exclude Subfolders" option (see bug #984), the highlighted picture instantly shows up on the right side (good!), but the Missing Pictures list does update. This is pretty confusing. The picture is showing, so we know PTXprint can find it, yet the list says the picture is still missing. It took me a while to realize the list would only refresh if I hit the "Print" button -- but printing sometimes take a while and I'd rather not have to run the entire print process to verify the pictures are now working.

I have no simple solution. But maybe if PTXprint would auto refresh whenever the user messes with any of the picture location options, or maybe add a "refresh list" button in that notification area?

**Comment by mhosken** [comment]
We have no intention of changing the behaviour here. If someone fixes a missing picture, they know they've fixed the missing picture until they hit print. It's too much cost for very little gain.

**Comment by davidc86** [comment]
Yes. It makes sense that it is too much of a CPU cost to refresh the list immediately. 
But it is a bit weird for the user to fix the picture location so that one of the pictures shows up in the dialog, and yet the UI still tells him that all the pictures are missing. From a new user's perspective this is confusing. Maybe if the list just said it was "the last run", like this maybe: 

Missing Pictures (last run): ZYZ, XYZ, ... 
(Rerun for an updated list.)
